dummy-files/.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,14 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="PyPackageRequirementsInspection" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="ignoredPackages">
+        <value>
+          <list size="1">
+            <item index="0" class="java.lang.String" itemvalue="uvloop" />
+          </list>
+        </value>
+      </option>
+    </inspection_tool>
+  </profile>
+</component>
```

Comment:
Should all these `.idea` files need to be included still?

Reply:
nope, thanks for noticing it